api/envoy/extensions/filters/http/bandwidth_limit/v3alpha/bandwidth_limit.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.bandwidth_limit.v3alpha;
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/migrate.proto";
```

Comment:
this isnt currently used

Reply:
do you mean this is deprecated or it's not being used in this particular PR? Guessing you meant the former since I can't find the doc which used to have a mention about using this with '*alpha' protos.

Reply:
this proto file doesnt use it so it emits a warning in CI

the current docs have just been updated to remove reference to `migrate.proto` here - https://github.com/envoyproxy/envoy/blob/main/api/STYLE.md#adding-an-extension-configuration-to-the-api

Reply:
got it. removed these imports.